picasso/src/test/java/com/squareup/picasso/DeferredRequestCreatorTest.java

```diff
@@ -54,48 +50,13 @@
     initMocks(this);
   }
 
-  @Config(reportSdk = 12)
-  @Test public void initAddsAttachListenerWhichDefersLayoutListener() {
```

Comment:
These tests were for API 12 and newer so if we're bumping the min then all that should be required is removing the `@Config`. I'm fine leaving it at 14. :YOLO: